pkg/koordlet/resourceexecutor/executor.go

```diff
@@ -83,7 +106,7 @@ func (e *ResourceUpdateExecutorImpl) UpdateBatch(cacheable bool, updaters ...Res
 		}
 
 		for _, updater := range updaters {
-			isUpdated, err := e.updateByCache(updater)
+			isUpdated, err := e.updateByCache(updater, nil)
```

Comment:
Strangely, there are some updates that have events while some other updates do not. Could you please adopt the event recorder uniformly or provide some information about why should not?

Reply:
the `eventHelper` is not nil ,  when only passed from the `update` function at `/pkg/koordlet/runtimehooks/protocal/protocal.go` ,  cause I think it's where we have the very first create pod/container event ?  
please let me know  if  I got something wrong, thanks!

```
func update(e resourceexecutor.ResourceUpdateExecutor, updater resourceexecutor.ResourceUpdater, eventHelper *audit.EventHelper) error {
	_, err := e.Update(true, updater, eventHelper)
	if err != nil {
		return err
	}
	return nil
}
```

Reply:
I don't get your point. Could you please elaborate on what cgroup events you think need to be set?
Please note that cgroups set by CPUBurst, CPUSuppress, and other features can be different from the `very first set cgroups` set by the runtime hooks on cgroup files and values. For instance, the CPUBurst sets the pod cgroup's `cpu.cpu_burst_us` while no runtime hook does.